### DISCUSSION TITLE
Make spec domain agnostic

### DISCUSSION THREAD

**Discussion by rahul101001000** [Sunbird-RC/community#67] [Sunbird RC Technical Specification Feedback]
In the Terminology section [link](https://docs.sunbirdrc.dev/learn/technical-specification-draft#visibility-scope) the last two terms are education specific. Could they be made domain agnostic? i.e. Information Provider and Information User instead of Education Information Provider and Education Information User.

**Comment by parthlawate** [comment]
Updated. Good catch @rahul101001000 !
